src/pages/home.html

```diff
@@ -0,0 +1,1926 @@
+                        >
+                            <p class="text-lg font-semibold">درباره عکسخونه</p>
+                            <svg
+                                width="24"
+                                height="24"
+                                viewBox="0 0 24 24"
+                                fill="none"
+                                xmlns="http://www.w3.org/2000/svg"
+                            >
+                                <circle
+                                    cx="12"
+                                    cy="12"
+                                    r="9"
+                                    stroke="#626E94"
+                                    stroke-width="1.5"
+                                    stroke-linecap="round"
+                                    stroke-linejoin="round"
+                                />
+                                <path
+                                    d="M12 9L9 12M9 12L12 15M9 12H15"
+                                    stroke="#626E94"
+                                    stroke-width="1.5"
+                                    stroke-linecap="round"
+                                    stroke-linejoin="round"
+                                />
+                            </svg>
+                        </a>
+                    </div>
+                    <div class="relative z-30 flex items-center justify-center">
+                        <div
+                            class="absolute left-0 top-0 h-full w-full bg-black-pearl bg-opacity-40"
+                        ></div>
+                        <img
+                            src="/src/assets/images/flower.png"
+                            alt="عکس گل"
+                            class="h-[605px] w-[589px] object-fill object-center"
+                        />
+                        <div class="flex items-center justify-center">
+                            <img
+                                src="/src/assets/images/icons/play.png"
+                                alt="آیکون پخش"
+                                class="absolute left-[45.5%] z-30 -translate-x-0.5 translate-y-0.5"
+                            />
+                            <div
+                                class="absolute left-[41%] z-20 h-[83px] w-[83px] -translate-x-0.5 translate-y-0.5 rounded-full bg-white outline outline-offset-[18px] outline-white"
+                            ></div>
+                            <div
+                                class="absolute left-[40%] z-10 h-24 w-24 -translate-x-0.5 translate-y-0.5 rounded-full bg-white bg-opacity-30 backdrop-blur-[42px]"
+                            ></div>
+                            <div
+                                class="absolute left-[36%] z-10 h-[138px] w-[138px] -translate-x-0.5 translate-y-0.5 rounded-full border border-white border-opacity-20"
+                            ></div>
```

Comment:
It is a good idea to implement all layers via HTML instead of using a single image. Because it can be animated or changed the color. But it's better to put all absolute elements in a single relative element it can be inside another absolute element.

By the way. In this case, do not need to put the relative element inside an absolute element to put it in the center of the parent. You can simply use the image as the background of the parent and make it flex center. Then put the relative element inside it.